imagepipeline/src/main/java/com/facebook/imagepipeline/producers/HttpUrlConnectionNetworkFetcher.java

```diff
@@ -34,6 +34,8 @@
   public static final int HTTP_TEMPORARY_REDIRECT = 307;
   public static final int HTTP_PERMANENT_REDIRECT = 308;
 
+  public static final int HTTP_DEFAULT_TIMEOUT = 30000;
```

Comment:
Can you make this value configurable please? Either by adding a second constructor and/or adding a setter for this.
This allows people to customize the timeout depending on their needs.

Reply:
Of course, it makes sense to make it configurable to the users personal need.